### ISSUE TITLE
LicenseFinder fails build of "aruba" on jRuby

### ISSUE THREAD

**Issue by maxmeyer** [pivotal/LicenseFinder#324]
Hey @kdykeman,

we get [build failures](https://travis-ci.org/cucumber/aruba/jobs/256795365#L589) on Travis' jRuby due to a [dependency](https://travis-ci.org/cucumber/aruba/jobs/256795365#L621) of `LicenseFinder`. I'm still investigating to give you some more details.

~~~
Gem::Ext::BuildError: ERROR: Failed to build gem native extension.
current directory:
/home/travis/.rvm/gems/jruby-9.1.9.0/gems/yajl-ruby-1.3.0/ext/yajl
/home/travis/.rvm/rubies/jruby-9.1.9.0/bin/jruby -r
./siteconf20170724-6662-1vmnrij.rb extconf.rb
creating Makefile
~~~

~~~
In Gemfile:
  license_finder was resolved to 3.0.1, which depends on
    yajl-ruby
~~~

**Comment by kdykeman** [comment]
@maxmeyer we just released version 3.0.2 of LicenseFinder today which should remove the yajl-ruby dependency. It also has numerous fixes for other issues in 3.0.1. Give the new version a shot and re-open this issue if you're still having the same problem.

**Comment by maxmeyer** [comment]
@kdykeman Great! I just started the build. This will take a while. Thanks!
